### ISSUE TITLE
Mange Preferences 404

### ISSUE THREAD

**Issue by sharongraylac** [stcr/subscribe-to-comments-reloaded#729]
Hey there! 

I'm still getting a 404 message when I click on "Manage Subscriptions," even after following the instructions for setting up a custom management page. Any thoughts on why this is still happening? 

Thanks!
Sharon

**Comment by mebishalnapit** [comment]
Could you provide the link to your site as well as the screenshot of the settings done for it for your site?

**Comment by sharongraylac** [comment]
Absolutely. The site is bellyandsoul.com, and here is a screenshot of my settings: https://snipboard.io/ro6CqA.jpg 

Thank you!

**Comment by mebishalnapit** [comment]
While checking your site, the comment subscriptions page seems to be working fine. So, could you check on it once properly and confirm? Maybe to check it, you can clear your browser cache and try to see if the issue still persists.

**Comment by sharongraylac** [comment]
I'm still getting a 404. It's triggered when I'm logged in and I click the "Manage Your Subscriptions Here" link.  Thanks!

**Comment by sharongraylac** [comment]
Sent. Thank you! :)

<http://thenourishedcook.com>
--
Sharon Gray, LAc
Belly & Soul <https://bellyandsoul.com>
